auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -312,6 +336,12 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
     window.location.href = appendAccountId(ConfigHelper.getNameRequestUrl())
   }
 
+  // create a numbered company
+  async startNumberedCompany () {
+    await this.createNumberedBusiness(this.currentAccountSettings.id)
```

Comment:
The source of the 10 numbered ben drafts

Reply:
This should be protected unless it's desirable and safe to call from outside this class.

Reply:
Updated I don't think there is any reason why it would need to be called from outside this class

Reply:
My comment is resolved (but I didn't start this conversation so I won't resolve it).